FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -710,3 +716,18 @@ def __storeProfiling():
             return result
     gMonitoringReporter.commit()
     return S_OK( 'Profiling information logged correctly' )
+  
+  @staticmethod
+  def __deleteOldSoftware( keepLast ):
+    """
+    It removes all versions except the last x
+    :param int keepLast: the number of the software version, what we keep
+    """
+    versionsDirectory = os.path.split( DIRAC.rootPath )[0]
+    softwareDirs = sorted( shutil.os.listdir( versionsDirectory ) )
```

Comment:
But it is not guaranteed that the shutil implementation will not remove the `import os`, is it?